compute_tools/src/compute.rs

```diff
@@ -1367,6 +1369,39 @@ LIMIT 100",
         download_size
     }
 
+    pub fn set_role_grants(
+        &self,
+        db_name: &str,
+        schema_name: &str,
+        privileges: &[Privilege],
+        role_name: &str,
+    ) -> Result<()> {
+        let mut conf = Config::from_str(self.connstr.as_str()).unwrap();
+        conf.dbname(db_name);
+
+        let mut db_client = conf
+            .connect(NoTls)
+            .context("Failed to connect to the database")?;
+
+        let query = "GRANT $1 ON SCHEMA $2 TO $3";
```

Comment:
does this work? I have my doubts

Reply:
right, guess we need to make it just a formatted string, then